### ISSUE TITLE
[Docs] Tutorials should use correct normalization values for image datasets

### ISSUE THREAD

**Issue by crypdick** [ray-project/ray#50239]
### Description

It is important for image datasets to use correct mean and standard deviation values ([more info here](https://www.richarddecal.com/data-science/use-these-normalization-values-for-torchvision-datasets/)). The Ray Docs often use inappropriate values for image datasets. For instance, [this tutorial](https://github.com/ray-project/ray/blob/27e58508950d5ff2a0cb042e46abc484ee4df912/doc/source/train/getting-started-pytorch-lightning.rst#L4) uses a mean and standard deviation of `0.5`, which are completely wrong for the FashionMNIST dataset.
